config/initializers/flipper.rb

```diff
@@ -1,7 +1,13 @@
 Flipper.configure do |config|
   config.default do
-    # pick an adapter, this uses memory, any will do
-    adapter = Flipper::Adapters::ActiveRecord.new
+    adapter = if (cache_duration = ENV["CACHE_FEATURE_FLAGS_SECONDS"])
+                Flipper::Adapters::ActiveSupportCacheStore.new(
+                  Flipper::Adapters::ActiveRecord.new,
+                  ActiveSupport::Cache::MemoryStore.new(expires_in: cache_duration.to_i.seconds),
+                )
+              else
+                Flipper::Adapters::ActiveRecord.new
+              end
```

Comment:
I wish Rubocop didn't rewrite this. I think it looked better when the assignment was done inside the `if` blocks and it made the whitespace-ignored diff easier to understand.